src/main/java/appeng/blockentity/networking/CableBusBlockEntity.java

```diff
@@ -346,7 +347,20 @@ public InteractionResult disassembleWithWrench(Player player, Level level, Block
         }
 
         // Play a break sound
-        level.playSound(player, getBlockPos(), SoundEvents.STONE_BREAK, SoundSource.BLOCKS, .7f, 1f);
+        BlockPos blockPos = hitResult.getBlockPos();
+        var facades = getFacadeContainer();
+        for (var side : Direction.values()) {
+            var facade = facades.getFacade(side);
+            if (facade != null) {
+                ItemStack masqItem = facade.getTextureItem();
+                BlockItem blockItem = (BlockItem) masqItem.getItem();
+                BlockState blockState = blockItem.getBlock().defaultBlockState();
+                SoundType soundType = blockState.getSoundType();
+
+                this.level.playLocalSound(blockPos, soundType.getBreakSound(), SoundSource.BLOCKS,
+                        1.0f, 1.0f, false);
+            }
+        }
```

Comment:
Your first comment is correct. I had misunderstood the purpose of that function

Reply:
To correct the issue I think it would be best to move dissasembly sounds to WrenchHook and add a general sound for facade or interface disassembly, like the sound an item frame makes when you remove an item from it.